localstripe/resources.py

```diff
@@ -943,6 +943,15 @@ def _api_list_subscriptions(cls, id, **kwargs):
 
         return cls._api_retrieve(id).subscriptions
 
+    @classmethod
+    def _api_list_sources(cls, id, object=None, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        # TODO: filter by "object=card"
```

Comment:
it bugged me so i implemented the filter ;-) it's not a 100% correct because cards and bank_account objects are returned differently, but at least it doesn't break the stripe client anymore.